docs/multicluster/policy-only-mode.md

```diff
@@ -0,0 +1,74 @@
+# Antrea Multi-cluster with NetworkPolicy Only Mode
+
+When Antrea is deployed with `networkPolicyOnly` mode, Antrea multi-cluster
+will be responsible for cross-clusters traffic only, any local cluster traffic
+will still be routed by primary CNI.
+
+## Deploying Antrea in `networkPolicyOnly` mode with Multi-cluster feature
+
+This document describes steps to deploy Antrea in `networkPolicyOnly` mode
+with multi-cluster feature enabled for EKS clusters.
+
+This document assumes you already have an EKS cluster, and have `KUBECONFIG`
+environment variable point to the kubeconfig file of that cluster. You can
+follow [the EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html)
+to create the cluster.
+
+You should apply `antrea-eks-node-init.yaml` before deploying Antrea.
+This will restart existing Pods (except those in host network), so that
+Antrea can also manage them once it is installed.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea-eks-node-init.yml
+```
+
+To deploy a released version of Antrea, pick a deployment manifest from the
+[list of releases](https://github.com/antrea-io/antrea/releases).
+Note that Antrea multi-cluster with `networkPolicyOnly` support was added
+in release v1.11.0, which means you cannot pick a release older than v1.11.0.
+The Antrea version is set to an environment variable `TAG`. For example,
+the following command sets the Antrea version to `v1.11.0`.
+
+```bash
+export TAG=v1.11.0
+```
+
+You need to get the manifest from the release page first, then set multi-cluster
+configurations before deployment.
+
+```bash
+curl -LO https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-eks.yml
+```
+
+You should set the following configuration parameters in `antrea-agent.conf` of
+the Antrea deployment manifest to enable the `Multicluster` feature:
+
+```yaml
+antrea-agent.conf: |
+...
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
```

Comment:
The latest antrea-agent.conf in `antrea.yml` looks like this:
```yaml

    multicluster:
    # Enable Antrea Multi-cluster Gateway to support cross-cluster traffic.
    # This feature is supported only with encap mode.
      enableGateway: false
    # The Namespace where Antrea Multi-cluster Controller is running.
    # The default is antrea-agent's Namespace.
      namespace: ""
```

Should we change the configuration in the doc to the latest antrea-agent configuraton version ?

Reply:
Will do, thanks.